docs/reference.md

```diff
@@ -333,6 +333,11 @@ $ curl -u USERNAME:PASSWORD https://openwhisk.ng.bluemix.net/api/v1/namespaces/w
 ]
 ```
 
+The API of OpenWhisk also supports asynchronous calls on websites from the client.
+To allow these calls on the client side, we respond to OPTIONS calls with a list of allowed Headers and allowed Hosts.
+The responses of the OpenWhisk-calls also include the header `Access-Control-Allow-Origin = *`
+
+**We do not recommend using this feature, because your authentication needs to be sent to OpenWhisk too. As result, you're credentials are on remote javascript clients and you do not have control over you're credentials anymore. An attacker would have the full control over your OpenWhisk account.**
```

Comment:
**As OpenWhisk supports only one key per account currently, it is not recommended to use CORS beyond simple experiments currently. Your key would need to be embedded in a webapplication's code, thus being accessible by every viewer of that page. Use with caution.**